src/DoctrineAuditBundle/Resources/views/Audit/entity_history.html.twig

```diff
@@ -10,6 +10,10 @@
             <th>Object Id</th>
             <th>User Id</th>
             <th>Username</th>
+            {% if show_user_firewall %}
```

Comment:
There's a typo here: should be `show_firewall_info` instead of `show_user_firewall `

Reply:
You're right, I'll fix that.